desktop/apps/auction2/templates/my_active_bids.jade

```diff
@@ -2,37 +2,33 @@ include ../../../components/bid_status/index
 
 if myActiveBids && myActiveBids.length
   h2 Your Active Bids
-  table
```

Comment:
No significant changes here, just converting from `table` to flexbox resulted in a big diff.